### PR TITLE
Fix inverse condition for PHP_VERSION_ID

### PR DESCRIPTION
**Pull request by szepeviktor**

### CODE REVIEW

File: wp-sentry.php [hunk 1]
```diff
@@ -19,7 +19,7 @@
 }
 
 // Make sure the PHP version is at least 7.1.
-if ( ! defined( 'PHP_VERSION_ID' ) && PHP_VERSION_ID < 70100 ) {
+if ( defined( 'PHP_VERSION_ID' ) && PHP_VERSION_ID < 70100 ) {
```
Comment (stayallive) [inline wp-sentry.php:22]: Interesting, this is indeed not doing what it should, what do you think about this suggestion though?

```suggestion
if ( ! defined( 'PHP_VERSION_ID' ) || PHP_VERSION_ID < 70100 ) {
```

According to the [PHP docs](https://www.php.net/manual/en/reserved.constants.php#reserved.constants.core) this constant was introduced in PHP 5.2.7 (and WordPress even has a [polyfill](https://github.com/WordPress/WordPress/blob/5ce95b7c8801764de85c6c8425a758d5b50912e2/wp-includes/random_compat/random.php#L32-L42) for it), however, you never know...

Comment (szepeviktor) [reply wp-sentry.php]: Done.

**Comment by szepeviktor** [comment]
Please consider using @phpstan, it has many more eyes than we humans. 👀

**Comment by szepeviktor** [comment]
> According to the PHP docs this constant was introduced in PHP 5.2.7

The thing is :) wp-sentry needs PHP 7.1

https://github.com/stayallive/wp-sentry/blob/06cb79c13953008359346c89772d35bcef52e1e0/readme.txt#L6

**Comment by stayallive** [comment]
> The thing is :) wp-sentry needs PHP 7.1

Yes. I'm not sure how much you are in the WordPress space, but WordPress let's you install any plugin and does not enforce that PHP requirement, it just shows it as a hint. So that's why this check is in here and I try to not crash that check if possible (although considering PHP 5.2 is probably overkill).

**Comment by szepeviktor** [comment]
> it just shows it as a hint

Then that hint is misleading as wp-sentry does support lower versions.
Please consider advertising the real lowest supported version in `readme.txt`

**Pull request merged**